examples/android/LuminaDemo/README.md

```diff
@@ -0,0 +1,18 @@
+# Instructions to re-create the setup
+
+- Create a new kotlin project in xcode
+- make sure you have aarch64-apple-ios target installed + simulator
```

Comment:
do we need apple-ios and simulator for android?

Reply:
Nope, leftover from copied readme